#!/usr/bin/python3
def raise_exception():
    print("Exception raised")
